conf/kops/helmfile.yaml

```diff
@@ -0,0 +1,542 @@
+    - name: server.replicaCount
+      value: {{ coalesce (env "CHART_REPO_SERVER_REPLICA_COUNT") "2" }}
+
+    ## Optional: CHART_REPO_IMAGE_TAG; e.g. v0.4.2
+    - name: server.image.tag
+      value: {{ env "CHART_REPO_IMAGE_TAG" }}
+
+    ## Optional: CHART_REPO_STORAGE; e.g. amazon
+    - name: server.env.open.STORAGE; e.g. amazon
+      value: {{ coalesce (env "CHART_REPO_STORAGE") "amazon" }}
+
+    ## Required: CHART_REPO_STORAGE_AMAZON_BUCKET; e.g. cp-staging-chart-repo
+    - name: server.env.open.STORAGE_AMAZON_BUCKET
+      value: {{ env "CHART_REPO_STORAGE_AMAZON_BUCKET" }}
+
+    ## Optional: CHART_REPO_STORAGE_AMAZON_PREFIX; e.g. "/"
+    - name: server.env.open.STORAGE_AMAZON_PREFIX
+      value: {{ coalesce (env "CHART_REPO_STORAGE_AMAZON_PREFIX") "/" }}
+
+    ## Required: CHART_REPO_STORAGE_AMAZON_REGION; e.g. us-west-2
+    - name: server.env.open.STORAGE_AMAZON_REGION
+      value: {{ coalesce (env "CHART_REPO_STORAGE_AMAZON_REGION") (env "AWS_REGION") }}
+
+    ## Optional: CHART_REPO_DEBUG; e.g. true
+    - name: server.env.open.DEBUG
+      value: {{ coalesce (env "CHART_REPO_DEBUG") "true" }}
+
+    - name: server.env.open.DISABLE_API
+      value: true
+
+    - name: server.env.open.ALLOW_OVERWRITE
+      value: false
+
+    ## Optional: CHART_REPO_SERVER_BASIC_AUTH_USER; e.g. server
+    - name: server.env.secret.BASIC_AUTH_USER
+      value: {{ coalesce (env "CHART_REPO_SERVER_BASIC_AUTH_USER") "server" }}
+
+    ## Required: CHART_REPO_SERVER_BASIC_AUTH_PASS; e.g. A11FF4E0-69A4-4F0F-9B74-A586A3C563C2
+    - name: server.env.secret.BASIC_AUTH_PASS
+      value: {{ env "CHART_REPO_SERVER_BASIC_AUTH_PASS" }}
+
+    ## Required: CHART_REPO_STORAGE_AWS_IAM_ROLE; e.g. cp-staging-chart-repo
+    - name: server.replica.annotations.iam\.amazonaws\.com/role
+      value: {{ env "CHART_REPO_STORAGE_AWS_IAM_ROLE" }}
+
+################################################################################
+## External DNS ################################################################
+################################################################################
+
+#
+# References:
+#   - https://github.com/kubernetes/charts/blob/master/stable/external-dns/values.yaml
+#
+
+- name: external-dns
+  namespace: kube-system
+  labels:
+    name: external-dns
+    namespace: kube-system
+    component: ingress
+    required: true
+  chart: stable/external-dns
+  version: 0.5.4
+  set:
+    - name: nodeSelector.kubernetes\.io/role
+      value: master
+
+    - name: extraEnv.EXTERNAL_DNS_SOURCE
+      value: service\ningress"
+
+    - name: tolerations[0].key
+      value: node-role.kubernetes.io/master
+
+    - name: tolerations[0].effect
+      value: NoSchedule
+
+    ## Required: EXTERNAL_DNS_TXT_OWNER_ID; e.g. 11591833-F9CE-407C-8519-35A947DB1D87
+    - name: txtOwnerId
+      value: {{ env "EXTERNAL_DNS_TXT_OWNER_ID" }}
+
+    ## Required: EXTERNAL_DNS_TXT_PREFIX; e.g. us-west-2.staging.cloudposse.org
```

Comment:
Required: EXTERNAL_DNS_TXT_PREFIX; e.g. 11591833-F9CE-407C-8519-35A947DB1D87-
Let's add `-` at the end b/c it will be added to all TXT records (easier to read them)